datastore/integration_test.go

```diff
@@ -484,6 +484,57 @@ func testSmallQueries(ctx context.Context, t *testing.T, client *Client, parent
 		x()
 	}
 }
+func TestIntegration_FilterEntity(t *testing.T) {
+	ctx := context.Background()
+	client := newTestClient(ctx, t)
+	defer client.Close()
+
+	parent := NameKey("SQParent", "TestIntegration_Filters"+suffix, nil)
```

Comment:
quibble: this being a test, you can be even more terse in your variable naming. Up to you!